app/src/main/java/com/findmyrecycling/MainActivity.kt

```diff
@@ -55,12 +58,55 @@ class MainActivity : ComponentActivity() {
                 // A surface container using the 'background' color from the theme
                 Surface(color = MaterialTheme.colors.background,
                 modifier = Modifier.fillMaxWidth()) {
-                    RecycleSearch("Android", products)
+                    RecycleSearch("", products)
                 }
             }
         }
     }
 
+    @Composable
+    fun RecycleSearch(name: String, products: List<Product> = ArrayList<Product>()) {
```

Comment:
Since RecycleSearch() seems to be your main composable and is containing other composable, I moved it to the top.  This seems to be how it done in the examples in the documentation. 

https://developer.android.com/jetpack/compose/layering